RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1338,7 +1336,7 @@ boolean canYellowRay(monster target)
 	# Use this to determine if it is safe to enter a yellow ray combat.
 
 	// first, do any necessary prep to use a yellow ray
-	if((my_familiar() == $familiar[Crimbo Shrub]) || (!forbidFamChange($familiar[Crimbo Shrub]) && auto_have_familiar($familiar[Crimbo Shrub])))
+	if((my_familiar() == $familiar[Crimbo Shrub]) || (canChangeToFamiliar($familiar[Crimbo Shrub]) && auto_have_familiar($familiar[Crimbo Shrub])))
```

Comment:
can remove  `&& auto_have_familiar($familiar[Crimbo Shrub])` (and the extra level of brackets that will leave).

Reply:
can also remove (my_familiar() == $familiar[Crimbo Shrub])
because in any scenario where your familiar is already set to X canChangeToFamiliar(X) will return true.

Reply:
commit 58b61fc